### DISCUSSION TITLE
How to collapse Datagrid RowDetails

### DISCUSSION THREAD

**Discussion by arnyCrd** [AvaloniaUI/Avalonia#10327] [Q&A]
Hi,
I have a Datagrid defined in a TemplatedControl.
The Datagrid is using RowDetailsTemplate that are working perfectly fine.
However I would like to collapse the RowDetails by clicking again on the DataRow.
Any idea on how to do that ?
Many thanks :) !

**Comment by cxxtreasure** [comment]
hi,
i have a problem with using RowDetailsTemplate ,can i learn your this code? thanks
